data/distro.yml

```diff
@@ -203,10 +203,14 @@
   steps:
     - name: Install Flatpak
       text: "
-        <p>To install Flatpak, enable the ~amd64 keyword for sys-apps/flatpak, acct-user/flatpak and acct-group/flatpak:</p>
-        <terminal-command>echo -e 'sys-apps/flatpak ~amd64\\nacct-user/flatpak ~amd64\\nacct-group/flatpak ~amd64\\ndev-util/ostree ~amd64' >> /etc/portage/package.accept_keywords/flatpak</terminal-command>
+        <p>To install Gentoo's stable version of Flatpak run:
+        <terminal-command>emerge --ask --sys-apps/flatpak</terminal-command>"
+    - name: Install Flatpak (testing version)
```

Comment:
These are steps to install it. 

I don't think it makes sense to have a test install step.

Kinda weird to have:

1. Install Flatpak
2. Install Flatpak test version

Reply:
I agree. Installing test version does not make sense for much regular users and more skilled Gentoo users are already familiar with unmasking and installing testing packages.

Reply:
I was going for a little bit of advertising the power of Gentoo to the masses while helping those that run into issues be able to be able to use a newer version.

Can drop if you disagree with the idea.

Reply:
That makes perfect sense, thanks for explaining.